core/templates/dev/head/components/ExplorationSummaryTileDirective.js

```diff
@@ -35,7 +35,8 @@ oppia.directive('explorationSummaryTile', [function() {
       getThumbnailBgColor: '&thumbnailBgColor',
       // If this is not null, the new exploration opens in a new window when
       // the summary tile is clicked.
-      openInNewWindow: '@openInNewWindow'
+      openInNewWindow: '@openInNewWindow',
+      isCommunityOwned: '=?isCommunityOwned'
```

Comment:
Can we make this a mandatory field (isCommunityOwned, &isCommunityOwned) and specify it everywhere there's a summary tile?